Editorial review: Document WebXR visibilitymaskchange event (#43975)
mdn/content: files/en-us/web/api/xrvisibilitymaskchangeevent/eye/index.md

---
title: "XRVisibilityMaskChangeEvent: eye property"
short-title: eye
slug: Web/API/XRVisibilityMaskChangeEvent/eye
page-type: web-api-instance-property
browser-compat: api.XRVisibilityMaskChangeEvent.eye
---

{{APIRef("WebXR Device API")}}{{SecureContext_Header}}

> The read-only **`eye`** property of the {{domxref("XRVisibilityMaskChangeEvent")}} interface indicates the eye the mask applies to.

Reviewer: For all
```suggestion
The **`eye`** read-only property of the {{domxref("XRVisibilityMaskChangeEvent")}} interface indicates the eye the mask applies to.
```

## Value

An emumerated value indicating which eye the mask applies to, which can be one of:

- `left`
  - : The {{domxref("XRView")}} represents the point-of-view of the viewer's left eye.
- `right`
  - : The view represents the viewer's right eye.
- `none`

>   - : The `XRView` describes a monoscopic view, or the view otherwise doesn't represent a particular eye's point-of-view.

Reviewer: There is a bunch of stuff that confuses me here. How does  {{domxref("XRView")}}  map to eye? Mentioning a view below feels like he wrong place - if there is a relationship then push it into the intro above or in the intro sentence here. 
This bit should look like

```suggestion
An emumerated value indicating which eye the mask applies to from the viewer's perspective.
This can be one of:

- `left`
  - : The viewer's left eye.
- `right`
  - : The viewer's right eye.
- `none`
  - : A monoscopic view, or the view otherwise doesn't represent a particular eye's point-of-view.
```

Author: Yeah, I think this is fine without mentioning `XRView`.

## Examples

> 

Reviewer: ```suggestion

### Basic usage

```
